<launch>
	<arg name="doDebug" default="false" />
	<arg if="$(arg doDebug)" name="launchPrefix" value="xterm -e gdb -args" />
	<arg unless="$(arg doDebug)" name="launchPrefix" value="" />
	<node name="line_detector_node" pkg="line_detector"  type="line_detector_node" output="screen" launch-prefix="$(arg launchPrefix)">
  	</node>
</launch>
